docs/eventing/event-registry.md

```diff
@@ -6,30 +6,30 @@ type: "docs"
 
 ## Overview
 
-The Event Registry maintains a catalog of the event types that can be consumed
-from the different Brokers. It introduces a new
-[EventType](../reference/eventing/) CRD in order to persist the event
-type's information in the cluster's data store.
+The event registry maintains a catalog of event types that can be consumed
+from different brokers. It introduces the [EventType custom resource](https://github.com/knative/eventing/blob/master/pkg/apis/eventing/v1beta1/eventtype_types.go) in order to persist the event
```

Comment:
I've heard discussions about avoiding linking users to Go code since they might not have familiarity with it. wdyt?

Reply:
I think that's a valid point, but this at least is better than a b0rk3n link? Maybe create an issue to make a larger effort to get away from linking to .go code and then deal with those in pieces?

Reply:
Yes, I totally agree.

Reply:
I opened https://github.com/knative/docs/issues/3309